gef-ui/server/index.js

```diff
@@ -5,7 +5,6 @@ const redis = require('redis');
 
 const flash = require('connect-flash');
 const path = require('path');
-const json2csv = require('express-json2csv');
```

Comment:
looks like this could be removed from package.json now?

Reply:
I believe this has been removed